src/drawing/__tests__/DrawingList-test.tsx

```diff
@@ -12,19 +13,22 @@ describe('DrawingList', () => {
     const getDefaults = (): Props => ({
         annotations: annotations as AnnotationDrawing[],
     });
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<DrawingList {...getDefaults()} {...props} />);
+    const getWrapper = (props = {}): ReactWrapper => mount(<DrawingList {...getDefaults()} {...props} />);
 
     beforeEach(() => {
         jest.spyOn(React, 'useState').mockImplementation(() => [null, jest.fn()]);
     });
 
     describe('render', () => {
-        test.each([true, false])('should render the class based on isListening %s', isListening => {
+        test.each([
+            { className: 'is-listening', isListening: true },
+            { className: '', isListening: false },
+        ])('should render the class based on isListening $isListening', ({ className, isListening }) => {
             (useIsListInteractive as jest.Mock).mockReturnValue(isListening);
 
             const wrapper = getWrapper();
 
-            expect(wrapper.hasClass('is-listening')).toBe(isListening);
+            expect(wrapper.find(DrawingSVG).prop('className')).toBe(className);
```

Comment:
Needed to refactor this as the test broke when I switched to using `mount` instead of `shallow`.